zk/lock.go

```diff
@@ -23,6 +24,8 @@ type Lock struct {
 	seq      int
 }
 
+var mapEphermeralSequenceLockPath = make(map[string]string)
```

Comment:
please specify in the comments what is the key and what is the value so that everyone understands it

Reply:
Done. Please check